cloudevents/tests/test_event_from_request_converter.py

```diff
@@ -25,7 +25,7 @@
 @pytest.mark.parametrize("event_class", [v03.Event, v1.Event])
 def test_binary_converter_upstream(event_class):
     m = marshaller.NewHTTPMarshaller([binary.NewBinaryHTTPCloudEventConverter()])
-    event = m.FromRequest(event_class(), data.headers[event_class], None, lambda x: x)
+    event = m.FromRequest(event_class(), data.headers[event_class], b"", lambda x: x)
```

Comment:
Using `None` is a type error, but in practice the value is a placeholder and never read. Using empty bytes is the correct type and acts as a similar placeholder value.